assets/sass/modules/_consent.scss

```diff
@@ -33,9 +33,18 @@
     left: 59px;
 
     .client-logo {
+      border-radius: $border-radius-default;
+    }
+
+    .custom-logo {
       width: 70px;
       height: 70px;
-      border-radius: $border-radius-default;
+    }
+
+    .default-logo {
```

Comment:
just a question: why are both logos different size? Just curious on why we don't have a default logo that is the same size as any of the custom logos... (what you have works, but just want to know the reason behind this)

Reply:
Is because of the image of our gear logo has no "padding" in the image itself. The custom logo could be literally anything, so is on the customer side decide if the image will have "padding" or not.